src/controllers/handlers.ts

```diff
@@ -248,6 +248,13 @@ export const parcelRequestHandler = async (context: {
     return { status: 503, body: 'Not ready' }
   }
 
+  if (x !== null && y !== null) {
```

Comment:
can `x` or `y` be `null` here? if so, wouldn't that break down the road? I feel like this `if` is unnecessary because they can't be null at this point OR we need an `else` that also returns an error in case any of them is `null`.

Reply:
absolutely right, good catch! They can't be null at this point, they will be either a number or string (if they send null as param it will be a string) so I removed the `if` since it was unnecessary!